javascripts/discourse/components/welcome-link-banner.js

```diff
@@ -1,40 +1,61 @@
 import Component from "@ember/component";
```

Comment:
If you're using tracking props you probably want a glimmer component:

```suggestion
import Component from "@glimmer/component";
```